modules/checking-mco-status-certs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="checking-mco-status-certs_{context}"]
+= Viewing and interacting with certificates
+
+The following certificates are handled in the cluster by the Machine Config Controller (MCC) and can be found in the `ControllerConfig` resource:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
```

Comment:
Apart from those certificates, MCC handles the "image registry" certificates and the  "image registry user bundle" certificate as well.